editor/src/components/editor/store/editor-state.ts

```diff
@@ -1471,6 +1472,7 @@ export interface EditorState {
   collaborators: Collaborator[]
   sharingDialogOpen: boolean
   editorRemixConfig: EditorRemixConfig
+  propertiesUpdatedDuringInteraction: UpdatedProperties
```

Comment:
isn't this leftover?

Reply:
It's very much not a leftover, this prop is read by these functions https://github.com/concrete-utopia/utopia/blob/09086039ca30b2d337f1b424ce71355a0bbf980c/editor/src/components/canvas/plugins/style-plugins.ts#L77-L83 and written to here https://github.com/concrete-utopia/utopia/blob/09086039ca30b2d337f1b424ce71355a0bbf980c/editor/src/components/canvas/plugins/style-plugins.ts#L132-L138